src/main.ts

```diff
@@ -216,7 +216,7 @@ export class CustomSonosCard extends LitElement {
       }
       .players {
         flex: 0 0 45%;
-        max-width: 45%;
+        max-width: 700px;
```

Comment:
I'd like to keep this as is. There is an upcoming change that makes this configurable: https://github.com/johanfrick/custom-sonos-card/commit/12f8ed3b2434380f3e86db6ece531e32338ef000

Reply:
I see, this will be a great feature to have! No need for the max width to be implemented if it's going to be like that.

Reply:
:-) So until that is out, can you revert this line?

Reply:
Ow! Sure! Didn't know you meant that.

Reply:
Layout is now configurable with https://github.com/johanfrick/custom-sonos-card/releases/tag/v2.4.0

https://github.com/johanfrick/custom-sonos-card#layout